aten/src/ATen/native/native_functions.yaml

```diff
@@ -2655,6 +2715,18 @@
   dispatch:
     MkldnnCPU: mkldnn_linear_backward
 
+- func: mps_linear_backward_input(int[] input_size, Tensor grad_output, Tensor weight) -> Tensor
```

Comment:
If these two functions (the _input and _weights ones) are only used internally by `mps_linear_backward`, no need for them to be here.

Reply:
I see, how will we get those functions defined?

Reply:
Done.